src/zelos/memory.py

```diff
@@ -572,6 +575,9 @@ def map_file(
             prot=prot,
             shared=shared,
         )
+        hooks = self._hook_manager._get_hooks(HookType.MEMORY.INTERNAL_MAP)
+        for hook in hooks:
+            hook(0, address, size, None)
```

Comment:
Consider returning data in internal map hook, rather than `None`

Reply:
Caused some of our LTP tests to fail due to timeouts. Left some comments stating that we won't pass the "value" part of the memory callback due to time costs.